frameworks/Java/voovan/src/main/java/org/voovan/VoovanTFB.java

```diff
@@ -65,7 +66,7 @@ public void process(HttpRequest req, HttpResponse resp) throws Exception {
                 webServer.get("/json", new HttpRouter() {
                         public void process(HttpRequest req, HttpResponse resp) throws Exception {
                                 resp.header().put(HttpStatic.CONTENT_TYPE_STRING, HttpStatic.APPLICATION_JSON_STRING);
-                                resp.write(JSON.toJSON(msg, false, false));
+                                resp.write(TString.toAsciiBytes(JSON.toJSON(msg, false, false)));
```

Comment:
`msg` needs to be instantiated on each request. I missed this before, but that doesn't appear to be happening here. 

`Message msg = new Message(HELLO_WORLD_STR)` needs to happen here.

Reply:
ok, i fixed it, thanks